R/object_name_linters.R

```diff
@@ -46,14 +45,16 @@ object_name_linter <- function(styles = "snake_case") {
       "//SYMBOL_FORMALS"
     )
 
-    assignments <- xml2::xml_find_all(x, xpath)
+    assignments <- xml2::xml_find_all(xml, xpath)
 
     # Retrieve assigned name
-    nms <- strip_names(
-      as.character(xml2::xml_find_first(assignments, "./text()")))
+    nms <- xml2::xml_text(assignments, "./text()")
+
+    # exclude operators if they're only symbols (e.g. %+%), #615
+    nms <- strip_names(grep("^[`'\"]%[^[:alnum:]]+%[`'\"]$", nms, invert = TRUE, value = TRUE))
```

Comment:
I don't know how to cleanly do this grep-filter with `rex`, do I have to use `!re_matches`?

Reply:
no longer needed